student-store-ui/src/components/App/App.jsx

```diff
@@ -1,21 +1,85 @@
-import * as React from "react"
-import { BrowserRouter } from 'react-router-dom'
-import Navbar from "../Navbar/Navbar"
-import Sidebar from "../Sidebar/Sidebar"
-import Home from "../Home/Home"
-import "./App.css"
+import * as React from "react";
+import axios from "axios";
+
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { useEffect, useState } from "react";
+
+import "./App.css";
+
+import Navbar from "../Navbar/Navbar";
+import Sidebar from "../Sidebar/Sidebar";
+import Home from "../Home/Home";
+import Hero from "../Hero/Hero";
+import SubNavbar from "../SubNavbar/SubNavbar";
+import About from "../About/About";
+import Contact from "../Contact/Contact";
+import Footer from "../Footer/Footer";
+import ProductDetails from "../Product Details/ProductDetails";
 
 export default function App() {
+  const url = "http://localhost:3001/";
```

Comment:
This could be moved out of the App component to be made more clear it's a global constant, right after the imports.